authentication/middleware_test.py

```diff
@@ -9,11 +9,15 @@
 from authentication.middleware import SocialAuthExceptionRedirectMiddleware
 
 
+def dummy_get_response(request):  # pragma: no cover
+    return None
+
+
 def test_process_exception_no_strategy(rf, settings):
     """Tests that if the request has no strategy it does nothing"""
     settings.DEBUG = False
     request = rf.get(reverse("social:complete", args=("email",)))
-    middleware = SocialAuthExceptionRedirectMiddleware()
+    middleware = SocialAuthExceptionRedirectMiddleware(dummy_get_response)
```

Comment:
We did something similar [here](https://github.com/mitodl/mitxpro/pull/2867/files#diff-d9426d57470e8e2a4d7b9dfd7815ef8fcc1d495ea9382d60111d9388ea20216eR18). This applies to the other place as well.

```suggestion
    middleware = SocialAuthExceptionRedirectMiddleware(get_response=mocker.Mock())
```